### PR TITLE
Add first benchmark test for DataFrame methods

### PR DESCRIPTION
**Pull request by PokhodenkoSA**

### CODE REVIEW

File: sdc/tests/tests_perf/test_perf_df.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+# *****************************************************************************
+# Copyright (c) 2019, Intel Corporation All rights reserved.
```
Comment (densmirn) [inline sdc/tests/tests_perf/test_perf_df.py]: Please change year in the copyright.

File: sdc/tests/tests_perf/test_perf_df.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+# *****************************************************************************
+
+import pandas as pd
+import numpy as np
+
+import time
+import random
+
+import pandas
+import sdc
```
Comment (densmirn) [inline sdc/tests/tests_perf/test_perf_df.py]: As I see some of the imports are not used. So let's clean it up.

File: sdc/tests/tests_perf/test_perf_df.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+        full_input_data_length = sum(len(i) for i in input_data)
+        for data_length in self.total_data_length[data_name]:
+            base = {
+                "test_name": test_name,
+                "data_size": data_length,
+            }
+            data = perf_data_gen_fixed_len(input_data, full_input_data_length,
+                                           data_length)
+            test_data = pandas.DataFrame({column: data for column in ('A', 'B', 'C')})
```
Comment (densmirn) [inline sdc/tests/tests_perf/test_perf_df.py]: You could use `string.ascii_uppercase` from default module `string` to get all the letters and generate more big `DataFrame`. What do you think about that?

File: sdc/tests/tests_perf/test_perf_df.py (hunk 1 above)
Comment (PokhodenkoSA) [inline sdc/tests/tests_perf/test_perf_df.py]:
```suggestion
# Copyright (c) 2020, Intel Corporation All rights reserved.
```

File: sdc/tests/tests_perf/test_perf_df.py (hunk 2 above)
Comment (PokhodenkoSA) [inline sdc/tests/tests_perf/test_perf_df.py]:
```suggestion
```

File: sdc/tests/tests_perf/test_perf_df.py (hunk 3 above)
Comment (PokhodenkoSA) [inline sdc/tests/tests_perf/test_perf_df.py]:
```suggestion
            test_data = pandas.DataFrame({f"f{i}": data for i in range(3)})
```

**Pull request merged**